components/post_view/post_body_additional_content/post_body_additional_content.test.jsx

```diff
@@ -50,6 +50,27 @@ describe('components/post_view/PostBodyAdditionalContent', () => {
         });
     });
 
+    test('isLinkImage, should return false for tiffs', () => {
+        const testCases = [
+            {link: 'http://localhost/some_image1234.tiff', result: false},
```

Comment:
These could probably go with the other tests for `isLinkImage`, but that doesn't really matter much.

Reply:
@hmhealey you're right; I put these tests with the others in `isLinkImage`!